### PR TITLE
Darklight features

### PR DESCRIPTION
**Pull request by darklight05**
A quick bootstrap form. Gonna start looking for free APIs

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -1 +1,49 @@
...
+    </div>
+</div>
+
+
+<div id="appendResults">
+    <!-- A table or something -->
+    <!-- Dynamically manipulated in app.py -->
+</div>
+    
+<script src="app.py"></script>
```
Comment (mschrader15) [inline index.html:44]: Why are you referencing app.py? I don't think a browser can load python?

**Comment by mschrader15** [comment]
I doubt if there are free APIs. I was thinking that we would manually have to create the tables ourselves

File: index.html (hunk 1 above)
Comment (darklight05) [reply index.html:44]: Really? I thought they could

Comment (mschrader15) [reply index.html:44]: maybe.. `app.py` is the server anyways though so makes no sense to load in browser

**Pull request merged**